src/index.html

```diff
@@ -5,8 +5,343 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+          src="images/logo.png"
+          alt="MOYO"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#" class="is-active"> Apple </a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link"> Samsung </a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link"> Смартфоны и телефоны </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link"> Гаджеты </a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link"> Планшеты </a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link"> Фото </a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link"> Видео </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="main__card">
+        <!-- First card -->
+        <div class="card" data-qa="card">
+          <img class="card__image"
+              src="images/imac.jpeg"
+              alt="iMac A1419"
+          >
+          <div class="card__discription-container">
+            <h2 class="card__name-of-product">
+                Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </h2>
+            <p class="card__code-of-product">
+                Код товара: 195434
+            </p>
+          </div>
+          <div class="card__rating-container">
+            <ul class="stars">
```

Comment:
there should be modifier, like `stars--4`, responsible for amount of active stars